pkg/common/monitoring/monitoring.go

```diff
@@ -114,3 +116,27 @@ func GetClusterShardingStrategy(cluster metav1.Object) (*sharding.Strategy, erro
 		ScaleDownPercentage: scaleDownPercentage,
 	}, nil
 }
+
+func ListTenants(k8sClient client.Client, ctx context.Context) ([]string, error) {
```

Comment:
I think usually, we go ctx then other args

Reply:
Oh ok my bad, fixing this.